modules/150-user-authn/crds/doc-ru-dex-authenticator.yaml

```
@@ -87,30 +87,30 @@ spec:
                     properties:
                       effect:
                         description: |
-                          Определяет, какому effect'у taint'а соответствует описываемый toleration. Пустой подразумевает соответствие любому effect'у.
+                          Указывает, к какому значению `effect` у `taint` должен подходить данный `toleration`. Если не задан, `toleration` будет соответствовать любому `effect`.
```

```suggestion
                          Указывает, к какому значению `effect` у taint должен подходить данный toleration. Если не задан, toleration будет соответствовать любому `effect`.
```
Перегруз бэктиков. Давай попробуем уменьшить